02LessonTwo/in-class-project/index.html

```diff
@@ -1,10 +1,56 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Pups on Point!</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+<img src="http://www.activemindsdogtraining.com/images/Logo.jpg" id="logo"/>
+<h1><strong>Pups on Point</strong></h1>
+<h3>Perfecting puppies performance permanently!<h3>
+<br>
+<hr>
+<br><br>
+<pre>
+ <p>At Pups on Point we are commited to teaching your
+    fluffy beloved the basics of being an awesome,
+    well mannered friend!</p>
+</pre>
+<ul>
+  <li>Social skills</li>
+  <li>Energy Management</li>
+  <li>Impulse control</li>
+  <li>Leash manners</li>
+  <li>Separation Anxiety Treatment</li>
+  <li>Crate training</li>
+<br><br>
+</ul>
+<table>
+  <thead>
+    <th>obedience</th>
+    <th>Agility</th>
+    <th>Dancing</th>
+    </thead>
+  <tbody>
+    <tr>
+      <td>
+        <a href="https://www.youtube.com/watch?v=cmhGsDl0OWI">
+        <img src="http://www.advancedcanine.com/wp-content/uploads/2016/05/home-programs.jpg" class="service"/>
+      </a>
+      </td>
+      <td>
+        <a href="https://www.youtube.com/watch?v=eK7rVv1TZtQ">
+        <img src="http://companiondogtrainingclub.com/wp-content/uploads/2014/04/dog-ttraining.jpg" class="service"/>
+      </a>
+      </td>
+      <td>
+        <a href="https://www.youtube.com/watch?v=qBh25WDiSpY">
+        <img src="http://i.dailymail.co.uk/i/pix/2012/06/12/article-2157978-136B572F000005DC-202_634x500.jpg" class="service"/>
+      </a>
+      </td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
great work! dont forget to put contact information at the bottom of the page. CSS is outstanding.